src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -0,0 +1,156 @@
+      class="kuiLocalTitle"
+    >
+      Visualize
+    </div>
+  </div>
+</kbn-top-nav>
+
+<div class="kuiViewContent kuiViewContent--constrainedWidth">
+  <!-- ControlledTable -->
+  <div class="kuiViewContentItem kuiControlledTable kuiVerticalRhythm">
+    <!-- ToolBar -->
+    <div class="kuiToolBar">
+      <div class="kuiToolBarSearch">
+        <div class="kuiToolBarSearchBox">
+          <div class="kuiToolBarSearchBox__icon kuiIcon fa-search"></div>
+          <input
+            class="kuiToolBarSearchBox__input"
+            type="text"
+            placeholder="Search..."
+            aria-label="Filter"
+            ng-model="listingController.filter"
+          >
+        </div>
+      </div>
+
+      <div class="kuiToolBarSection">
+        <!-- We need an empty section for the buttons to be positioned consistently. -->
+      </div>
+
+      <div class="kuiToolBarSection">
+        <!-- Bulk delete button -->
+        <button
+          class="kuiButton kuiButton--danger"
+          confirm-click="listingController.deleteSelectedItems()"
+          confirmation="Are you sure you want to delete the selected visualizations? This action is irreversible!"
+          aria-label="Delete selected objects"
+          ng-hide="listingController.getSelectedItemsCount() === 0"
+          tooltip="Delete selected visualizations"
+        >
+          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+        </button>
+
+        <!-- Create visualization button -->
+        <a
+          class="kuiButton kuiButton--primary"
+          href="#/visualize/step/1"
+          aria-label="Create new visualization"
+          ng-hide="listingController.getSelectedItemsCount() > 0"
+          tooltip="Create new visualization"
+        >
+          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-plus"></span>
+        </a>
+      </div>
+    </div>
+
+    <!-- NoResults -->
+    <div class="kuiPanel kuiPanel--centered" ng-if="!listingController.items.length">
+      <div class="kuiNoResults">
+        No visualizations matched your search.
+      </div>
+    </div>
+
+    <!-- Table -->
+    <table class="kuiTable" ng-if="listingController.items.length">
+      <thead>
+        <tr>
+          <th class="kuiTableHeaderCell kuiTableHeaderCell--checkBox">
+            <input
+              type="checkbox"
+              class="kuiCheckBox"
+              ng-checked="listingController.areAllItemsChecked()"
+              ng-click="listingController.toggleAll()"
+            >
+          </th>
+
+          <th class="kuiTableHeaderCell kuiTableHeaderCell--icon">
+          </th>
+
+          <th class="kuiTableHeaderCell">
+            Name
+          </th>
+
+          <th class="kuiTableHeaderCell">
+            Type
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr
+          ng-repeat="item in listingController.items track by item.id | orderBy:'title'"
```

Comment:
Good catch, this actually needs to be:

```
ng-repeat="item in listingController.items | orderBy:'title' track by item.id"
```

Relevant links:

* https://code.angularjs.org/1.4.14/docs/api/ng/filter/orderBy
* http://stackoverflow.com/questions/16455832/orderby-not-working-as-expected-angularjs

I'll make a PR to fix this in Dashboard too.